internal/test/test.go

```diff
@@ -617,3 +617,10 @@ func (h *Helper) GetCommit(repo string) string {
 	}
 	return strings.TrimSpace(string(out))
 }
+
+func (h *Helper) RemoveFile(path string) {
+	err := os.RemoveAll(path)
```

Comment:
If this is really just for removing a single file, then it should probably be reduced to just use `os.Remove()`